### PR TITLE
METEOR@1.6.0.1

### PR DESCRIPTION
**Pull request by aaronjudd**
METEOR 1.6.0.1

Node has been upgraded to version 8.9.3, an important security release.
The npm package has been upgraded to version 5.5.1.

### CODE REVIEW

File: .meteor/packages [hunk 1]
```diff
@@ -33,7 +33,7 @@ service-configuration@1.0.11
 mdg:validated-method
 shell-server@0.3.0
 dynamic-import@0.2.0
-standard-minifier-js@2.2.3
```
Comment (spencern) [inline .meteor/packages]: Any idea why it dropped the version for standard minifier?

Comment (aaronjudd) [reply .meteor/packages]: Looking into it..

Comment (aaronjudd) [reply .meteor/packages]: `The specified packages are at their latest compatible versions.`. Tested-docker builds still function and this was the main deal we had with the migration to this minifier.

Comment (spencern) [reply .meteor/packages]: ok, so we're good to merge you think?

Comment (aaronjudd) [reply .meteor/packages]: I do 👍

**Pull request merged**